.github/workflows/default.yaml

```diff
@@ -0,0 +1,92 @@
+name: Default
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [labeled]
```

Comment:
Only for labelled PRs?

Reply:
Yes, I tried to explain it in the PR description:
```The way to trigger CI is via the ci-test label. This avoids overloading the remote cache and remote build executors.```

Want me to add as a comment?

Reply:
Nope, just curious.